src/components/widgets/AutocompleteWidget/AutocompleteWidget.tsx

```diff
@@ -217,33 +232,79 @@ function AutocompleteWidget(props: AutocompleteWidgetProps) {
         }
     };
 
-    function onChangeHandler(options: EuiComboBoxOptionOption<any>[]): void {
+
+    function onChangeHandler(options: Array<any>): void {
         setSelectedOptions(options);
     }
 
-    useEffect(() => {
-    }, [selectedOptions, options, props.selectOption])
+    function onCreateOptionHandler(searchValue: string) {
+        const normalizedSearchValue = searchValue.trim().toLowerCase();
 
+        if (!normalizedSearchValue) {
+          return;
+        }
```

Comment:
You normalize the search value but don't use it...

Reply:
True - I removed it :+1: